submissions/MarharytaBoichenko/html_css_popup/sass/normalize.scss

```diff
@@ -0,0 +1,28 @@
+html {
+  box-sizing: border-box;
+  min-height: 100%;
+}
+*,
+*::after,
+*::before {
+  box-sizing: inherit;
+}
+
+h1,
+h2,
+h3,
+p {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+img {
+  display: block;
+  max-width: 100%;
+  height: auto;
+}
+
+a {
+  text-decoration: none;
+  color: currentColor;
+}
```

Comment:
An empty line is always needed at the end of the file
https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b